.pre-commit-config.yaml

```diff
@@ -104,13 +104,21 @@ repos:
           - --comment-style
           - <!--|   ~|  -->
       - id: insert-license
```

Comment:
Please split formatting changes in separate PR, unless there is an explicit reason not to (in case there is, please add to PR description)